x-pack/plugins/maps/common/constants.ts

```diff
@@ -31,6 +31,8 @@ export const CHECK_IS_DRAWING_INDEX = `/${GIS_API_PATH}/checkIsDrawingIndex`;
 export const MVT_GETTILE_API_PATH = 'mvt/getTile';
 export const MVT_GETGRIDTILE_API_PATH = 'mvt/getGridTile';
 export const OPEN_LAYER_WIZARD = 'openLayerWizard';
+export const SAVED_OBJECTS_LIMIT_SETTING = 'savedObjects:listingLimit';
```

Comment:
Would mind moving these constants into `maps/public/routes/list_page/maps_list_view.tsx` since they are only used in that file and do not need to be shared between server and client code

Reply:
Done!